tests/ltp/patches/fix_mkdir_mkdir02.patch

```diff
@@ -0,0 +1,74 @@
+	This change is necessary to run this test case in sgx-lkl
+	environment. Test case is modified to work in single
+	process environment by Removing exit code from child and 
+	wait code from parent. FORK_OR_VFORK return always 0 in
+	sgxlkl environment. Application in sgx-lkl environment
+	start with root user if this is the case Effective
```

Comment:
This change is necessary to run this test case in sgx-lkl
	environment. Test case is modified to work in single
	process environment by removing exit code from child and 
	wait code from parent. FORK_OR_VFORK always returns 0 in
	sgxlkl environment. Application in sgx-lkl environment
	starts with root user and some permissions lost when user
	changed from root to non-root. Hence, test case is updated
	to handle this scenario as well.

Reply:
I will update the patch description as per your comment.

Reply:
Done